git_cml/git_utils.py

```diff
@@ -142,55 +197,50 @@ def write_staged_file(f, contents):
             json.dump(contents, f, indent=4)
 
 
-def load_staged_file(f):
+def add_file(f, repo):
     """
-    Load staged file
+    Add file to git staging area
 
     Parameters
     ----------
-    f : str or file-like object
-        staged file to load
-
-    Returns
-    -------
-    dict
-        staged file contents
+    f : str
+        path to file
+    repo : git.Repo
+        Repo object for current git repository
     """
-    if isinstance(f, io.IOBase):
-        return json.load(f)
-    else:
-        with open(f, "r") as f:
-            return json.load(f)
+    logging.debug(f"Adding {f} to staging area")
+    repo.git.add(f)
 
 
-def write_staged_file(f, contents):
+def git_lfs_install():
     """
-    Write staged file
+    Run the `git lfs install` command
 
-    Parameters
-    ----------
-    f : str or file-like object
-        file to write staged contents to
-    contents : dict
-        dictionary to write to staged file
+    Returns
+    -------
+    int
+        Return code of `git lfs install`
     """
-    if isinstance(f, io.IOBase):
-        json.dump(contents, f, indent=4)
-    else:
-        with open(f, "w") as f:
-            json.dump(contents, f, indent=4)
+    out = subprocess.run(["git", "lfs", "install"])
+    return out.returncode
 
 
-def add_file(f, repo):
+def git_lfs_track(repo, directory):
```

Comment:
Maybe `git_lfs_track_directory` in case we later want a git LFS track function for files?